pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -170,8 +170,22 @@ in stdenv.mkDerivation {
     "-Dman=true"
   ];
 
-  preConfigure = ''
+  preConfigure = let
+    defaultDnsServers = [
+      # We use these public name services, ordered by their privacy policy (hopefully):
```

Comment:
I'm not sure if the ordering here actually means anything - it seems resolved picks one (or multiple) arbitrary server from the list:

https://systemd.network/resolved.conf.html#DNS=

Reply:
Well, given that fact I don't know how to proceed. Some may want to remove the google one?